lib/mp4box-player/src/Mp4boxPlayer.js

```diff
@@ -22,6 +22,9 @@ export class Mp4boxPlayer {
 
     this.mp4boxfile;
     this.movieInfo;
+    this.videoFps = 30; //Default value for fps to be used for segmentation samples 
+    this.desiredChunkSize = 409600; // Default 1MB chunksize
```

Comment:
I think this default valus is not 1MB or 4MB.
<img width="603" alt="image" src="https://github.com/stacknide/mp4box-apps/assets/60546840/4fe903b3-9494-49d8-baa0-39cbb2ce5f46">

Reply:
Comment is not updated. Chunk size finally tested to be working fine is 400KB, not 1MB